### PR TITLE
Use Docusaurus's default theme

### PR DESCRIPTION
**Pull request by jeangovil**

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -44,8 +44,54 @@ const config = {
...
+                href: "https://stackoverflow.com/questions/tagged/docusaurus",
+                label: "Stack Overflow",
+              },
+              {
+                href: "https://forum.vac.dev/",
+                label: "Discourse",
+              },
+              {
+                href: "https://discord.gg/j5pGbn7MHZ",
+                label: "Twitter",
```
Comment (fryorcraken) [inline docusaurus.config.js:73]:
```suggestion
                href: "https://discord.gg/Nrac59MfSX",
                label: "Discord",
```

File: docusaurus.config.js [hunk 2]
```diff
@@ -44,8 +44,54 @@ const config = {
...
+      },
+      footer: {
+        links: [
+          {
+            title: "Community",
+            items: [
+              {
+                href: "https://stackoverflow.com/questions/tagged/docusaurus",
+                label: "Stack Overflow",
+              },
```
Comment (fryorcraken) [inline docusaurus.config.js:66]:
```suggestion
```

**Review by fryorcraken — APPROVED** [review]
Thanks @jeangovil . Looks good.

@amirhouieh wants to host all Waku doc in this repo. Is it possible to have a language switcher? At first I need to be able to easily between the concept doc (current content) and the JS doc (https://github.com/waku-org/js.waku.guide).

**Comment by hackyguru** [comment]
is it a good idea to add the toggle button that switches between dark mode and light mode?

**Comment by fryorcraken** [comment]
> Is it a good idea to add the toggle button that switches between dark mode and light mode?

More importantly we need the language switch :)

**Pull request merged**